psi4/src/psi4/libscf_solver/uhf.cc

```diff
@@ -208,7 +207,7 @@ void UHF::form_G() {
     double alpha = functional_->x_alpha();
     double beta = functional_->x_beta();
 
-    if (alpha != 0.0) {
+    if ( alpha != 0.0 && !(functional_->is_x_lrc() && jk_->name() == "MemDFJK" && jk_->get_wcombine()) ){
```

Comment:
It might be best if `jk_->wcombine()` or similar is for every JK object which normally returns false. This could be useful for Direct as well.

Reply:
The memory checker does make the correct estimation based on this new operation.

As far as testing it goes, the only way would be to set the memory in the test script and to make sure that psi gives back the right value, but that could be problematic if someone messes with the fudge factor and gibi vs. giga bytes.

The complicated logic in that if statement is necessary because there are conditions that can be taken for granted without wcombine that can't without it. 

It would be useful for direct, but I wanted to wait on the libint2 branch to get sorted before I did anything with that.

Reply:
I'm not suggesting that you implement it into Direct at the moment, but more of the point that `jk_->name() == "MemDFJK" && jk_->get_wcombine()` is a leaky abstraction and creates a headache throughout the code. This `get_wcombine` should be native to the JK base object and return false for all classes where this isn't implemented.

Otherwise if you add this to direct we will need to add additional logic to every instance where you wrote this.

Reply:
`jk_->get_wcombine()` will already return false for all classes that aren't `MemDFJK` as you've suggested. I included the condition `jk_->name() == "MemDFJK"` because I didn't think I would need the `!jk_->get_wcombine()` case when I started working on the code.

I'm going to try getting rid of the `jk_->name() == "MemDFJK"` condition. If I understand the code correctly, it should work.

Reply:
I made the change and tested it. It seems to be working just fine.

Reply:
fyi, commit message would more precisely say "condition" than "conditional"